modules/rostests/apitests/ntdll/NtQueryInformationProcess.c

```diff
@@ -270,12 +257,14 @@ Test_ProcessWx86Information(void)
     ok_hex(Status, STATUS_INFO_LENGTH_MISMATCH);
 
     /* Given an invalid process handle */
+    VdmPower = 0xDEADBEEF;
     Status = NtQueryInformationProcess(NULL,
                                        ProcessWx86Information,
                                        &VdmPower,
                                        sizeof(VdmPower),
                                        NULL);
     ok_hex(Status, STATUS_INVALID_HANDLE);
+    ok_dec(VdmPower, 0xDEADBEEF);
```

Comment:
That doesn't make much sense to me. Whenever `NtQueryInformationProcess` fails for whatever reason, such as in this case, the submitted variable that had to get the queried information doesn't change simply because the function halted during the moment the failure has been constrained, unless you have discovered that Windows does something to that variable whilst the function bails out.